src/main/java/uk/gov/digital/ho/proving/income/audit/statistics/PassRateStatisticsService.java

```diff
@@ -35,6 +37,14 @@ public PassRateStatisticsService(AuditClient auditClient,
         this.requestPageSize = requestPageSize;
     }
 
+    public PassRateStatistics generatePassRateStatistics(YearMonth calendarMonth) {
+        return generatePassRateStatistics(calendarMonth.atDay(1), calendarMonth.atEndOfMonth());
+    }
+
+    public PassRateStatistics generatePassRateStatistics(TaxYear taxYear) {
```

Comment:
Have you got a set of tests for TaxYear args, in the same way there are for YearMonth args?

Reply:
Relies on the Tax Year to do calculation - testing interface to method